prov/efa/test/efa_unit_tests.c

```diff
@@ -10,6 +10,9 @@ static int reset_global_mocks(void **state)
 	g_efa_unit_test_mocks = (struct efa_unit_test_mocks) {
 		.ibv_create_ah = __real_ibv_create_ah,
 		.efadv_query_device = __real_efadv_query_device,
+#if HAVE_EFADV_CQ_EX
+		.efadv_cq_from_ibv_cq_ex = __real_efadv_cq_from_ibv_cq_ex,
```

Comment:
better always have this mock, but when `HAVE_EFADV_CQ_EX` is `0`, the function does not do anything.

Reply:
See response below.